treeherder/webapp/api/throttling.py

```diff
@@ -0,0 +1,17 @@
+from rest_framework import throttling
+
+
+class OauthKeyThrottle(throttling.ScopedRateThrottle):
+    def get_cache_key(self, request, view):
```

Comment:
The documentation doesn't describe the interface, so I looked directly at the code of rest_framework.throttling.ScopedRateThrottle